firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -696,7 +701,13 @@ def test_update_with_nulls_success_200(self):
 
         expected_data = self.add_expected_defaults(data)
         self.assert_updated_instance(expected_data, self.get_updated_instance(instance))
-        self.assert_update_response(expected_data, response_data)
+
+        # We need to pass instance to verify that the original is not modified
+        # in case of media files
+        if self.model in [Audio, Image, Video]:
+            self.assert_update_response_media(instance, expected_data, response_data)
+        else:
+            self.assert_update_response(expected_data, response_data)
```

Comment:
Same comment here about overriding it in a subclass rather that putting a type check in the parent